src/file.js

```diff
@@ -153,6 +153,23 @@
   };
 
   /**
+   * Method to replace an extension, if one does not exist in the file string, it will be added.
+   *
+   * @param {string} fileString - File string to add the extension to if one does not exist
+   * @param {string} extension - Extension to be either added to `fileString` or to replace the extension in `fileString`. The
+   *   value is the extension without the `.`. E.g. `js`, `html`.  Not `.js`, `.html`.
+   * @returns {string} fileString with the new extension
+   */
+  File.replaceExtension = function(fileString, extension) {
+    var regex = /([^.\/\\]+\.)[^.]+$/;
```

Comment:
No need to create a variable for this.  We can add it right in the `replace` call.  We can change this if the `regex` is used in more than one place